### DISCUSSION TITLE
Can't find jPlag via IntelliJ's Maven plugin

### DISCUSSION THREAD

**Discussion by onnimikki** [jplag/JPlag#1009] [Q&A] [answered]
Hello,

Strangely, when I try to search for de.jplag or jplag using the Maven plugin for IntelliJ, jPlag doesn't appear.  Usually the Maven tool in IntelliJ is really good at finding Maven projects, but I'm not sure why it's having problems with jPlag.  Does anyone have a suggestion for how to perform a successful search for jPlag on IntelliJ's Maven?

Thanks!

James

**Comment by tsaglam** [comment, marked as answer]
Sorry, I am not entirely sure what you mean. What happens if you manually set the [mvn dependencies](https://central.sonatype.com/search?smo=true&q=de.jplag)?

**Reply by tsaglam** [reply]
@onnimikki does the issue still persist?
